xdn-sapper-example/xdn/cache.js

```diff
@@ -0,0 +1,28 @@
+export const SSR_CACHE_HANDLER = ({ removeUpstreamResponseHeader, cache }) => {
+  removeUpstreamResponseHeader('cache-control')
+  cache({
+    browser: {
+      maxAgeSeconds: 0,
+    },
```

Comment:
```suggestion
    browser: false,
```

This is a bit more readable ^^